### ISSUE TITLE
 INFO: DO use curly braces for all flow control structures.

### ISSUE THREAD

**Issue by Luckey-Elijah** [Luckey-Elijah/coffee#5]
From [pub.dev](https://pub.dev/packages/coffee/score).

The location: `lib/src/classes/brew_recipe.dart:35:7`

```
   ╷
35 │       assert(grinder!.range!.isInRange(grindSize));
   │       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
```

**Comment by Luckey-Elijah** [comment]
https://github.com/Luckey-Elijah/coffee/blob/main/lib/src/classes/brew_recipe.dart#L35

**Comment by Luckey-Elijah** [comment]
fixed by c77a8b1
